std-bits/table/src/main/java/org/enso/table/data/column/operation/unary/AbstractUnaryBooleanOperation.java

```diff
@@ -18,21 +18,21 @@ protected AbstractUnaryBooleanOperation(String name, boolean nothingUnchanged) {
   }
 
   @Override
-  protected BoolBuilder createBuilder(
+  protected BuilderForBoolean createBuilder(
       ColumnStorage storage, MapOperationProblemAggregator problemAggregator) {
     if (storage.getSize() > Integer.MAX_VALUE) {
       throw new IllegalArgumentException(
           "Cannot currently operate on columns larger than " + Integer.MAX_VALUE + ".");
     }
-    return new BoolBuilder((int) storage.getSize());
+    return Builder.getForBoolean((int) storage.getSize());
   }
 
   @Override
   protected final void applyObjectRow(
       Object value, Builder builder, MapOperationProblemAggregator problemAggregator) {
-    applyObjectRow(value, (BoolBuilder) builder, problemAggregator);
+    applyObjectRow(value, (BuilderForBoolean) builder, problemAggregator);
```

Comment:
Again, I'd prefer to avoid such casts by usage of generic types.

I'm just not a fan of casts where we can use the type system to our advantage :)

But if you think it's overkill, I'm ok to skip that.

Reply:
This is unchanged from our current UnaryOperation but will revisit when I get to operations.

Reply:
Oh right, I haven't noticed 🤦 
Good to get it improved if we can in further iterations, but definitely totally fine the way it is now too.